code/modules/mob/mob_movement.dm

```diff
@@ -112,6 +112,10 @@
 		if(L.incorporeal_move)	//Move though walls
 			Process_Incorpmove(direct)
 			return 0
+	if(iscarbon(mob))
+		var/mob/living/carbon/C = mob
+		if(C.remote_view)

```

Comment:
Oh. Sure, i'll swap that around. Didn't notice something like this already existed